lib/models.js

```diff
@@ -14,17 +14,14 @@ var errors = require('./errors');
 
 //----------------------------------------------------------------------------------------------------------------------
 
-function Model(db, modelDef)
+function Model(modelDef)
 {
-    this.$$db = db;
-    this.$$values = {};
+    //TODO: Would it be safe to just make this _.clone() instead?
```

Comment:
Pretty sure it absolutely would be safe to use `_.clone`, since we're deep cloning what's being inserted into `$$defaultValues`.